test/setup.ts

```diff
@@ -9,6 +9,9 @@ if (!globalThis.fetch) {
   (globalThis as any).Response = Response;
 }
 
+// TODO: most tests rely on a spy on the global `console.log`. This means we must
+//    use silent: true here.  As an alternative we could explore using a `logger`
+//    that can be mocked, or spied on, or expose an extension api depending on the test.
 const lt = new LocalTableland({ silent: true });
```

Comment:
Just added a note here.  
If we create a "logger" abstraction, we could enable the tests to spy on the "logger" instead of the global console object.  That would allow us to see the network logs during tests.
If that seems useful, it could be included in another PR.

Reply:
I'm okay with adding a logger object, though I'm not sure I'd prioritize it, I'd leave that judgement up to you.